gridt/src/app/movements/movements.page.html

```diff
@@ -71,3 +66,9 @@ <h3>
 </ion-list>
 
 </ion-content>
+
+<ion-fab vertical="bottom" horizontal="end" slot="fixed" style="padding-right: 0.5rem;">
+  <ion-fab-button color="primary" routerLink="/add">
+    <ion-icon name="add"></ion-icon>
+  </ion-fab-button>
+</ion-fab> 
```

Comment:
Not sure why this is added again, It's a bit weird since this change has already been accepted in a previous PR (Thank you for that by the way).

Reply:
To be honest, since I've just started using git seriously, I wonder whether this has anything to do with the fact that I didn't merge your `master` branch with my fork. 

How can I ensure this doesn't happen again?

Reply:
Make sure to fork the development branch. We have recently started a shift from api-development to just development as the development branch.